coalib/misc/Constants.py

```diff
@@ -109,7 +110,16 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+# Checks for default_coafile in coala/coalib
+for file_lists in os.listdir(coalib_root):
+    for file in file_lists:
+        if file == 'default_coafile':
+            system_coafile = os.path.join(coalib_root, 'default_coafile')
+        else:
+            system_coafile = os.path.join(coalib_root, 'base_coafile')
+
+if system_coafile == os.path.join(coalib_root, 'default_coafile'):
+    logging.warning('Use of file named default_coafile is deprecated.')
 
```

Comment:
use `os.path.isfile(fname)` instead of for loop, and you can warn inside else block itself